packages/workspace/src/browser/quick-open-workspace.ts

```diff
@@ -63,12 +63,8 @@ export class QuickOpenWorkspace {
             try {
                 stat = await this.fileService.resolve(uri);
             } catch { }
-            if (!stat ||
-                !this.preferences['workspace.supportMultiRootWorkspace'] && !stat.isDirectory) {
-                continue; // skip the workspace files if multi root is not supported
-            }
-            if (uri.toString() === tempWorkspaceFile.toString()) {
-                continue; // skip the temporary workspace files
+            if (uri.toString() === tempWorkspaceFile.toString() || !stat) {
```

Comment:
This check will no longer work, since we no longer use the same name for all untitled workspaces. You should use `CommonWorkspaceUtils.isUntitledWorkspace()`.

Reply:
Thanks! I made the update.